clusterloader2/cmd/clusterloader.go

```diff
@@ -66,6 +67,7 @@ func validateClusterFlags() *errors.ErrorList {
 
 func initFlags() {
 	flags.StringVar(&clusterLoaderConfig.ReportDir, "report-dir", "", "Path to the directory where the reports should be saved. Default is empty, which cause reports being written to standard output.")
+	flags.BoolVar(&clusterLoaderConfig.SetUpPrometheus, "set-up-prometheus", false, "Whether to set-up the prometheus monitoring in the cluster.")
```

Comment:
Maybe we should call it "enable-prometheus-server" ?

[I prefer enable than set-up, and I think we should be clear that it's about the server]

Reply:
Done.